### ISSUE TITLE
Can't connect to some PeerTube instances

### ISSUE THREAD

**Issue by jeberger** [TeamNewPipe/NewPipe#3728]
NewPipe version 0.19.4 refuses to connect to some PeerTube instances. For instance, when I go to `Settings → Contents → PeerTube instances`, click on the `+` button and type "<https://video.ploud.fr/>", I get a message: "Cannot validate the instance" and the instance is not added to the list.

Other PeerTube instances work (eg. <https://hostyour.tv>)

**Comment by B0pol** [comment]
I can't reproduce

**Comment by gkeegan** [comment]
Is it possible these instances are being blocked by either your network or an adblock list on your phone?

**Comment by Stypox** [comment]
I can't reproduce either, though maybe this has something to do with certificates. For example, on my phone the default instance (i.e. framatube) gives the same error because of a missing certificate on my phone

**Comment by jeberger** [comment]
The same URL works in Firefox on the same phone and network, so this should rule out a network or adblock issue. Is there a way to check or get more info to see if it is a certificate issue?

**Comment by eladkarako** [comment]
the website's certificate looks fine and valid.

**Comment by Stypox** [comment]
For me it works well in browser, too. Maybe it has something to do with the availability of TLS protocols? Maybe some are disabled in NewPipe? @mqus

**Comment by mqus** [comment]
Yeah it could be that the page only supports TLS 1.3 or only TLS ciphersuites which are not supported by the android version.

@jeberger what is your android version?

**Comment by jeberger** [comment]
> @jeberger what is your android version?

It's the stock KitKat ROM on a Samsung Galaxy Note 3. Just tried with the stock browser and it can't connect either: it returns an SSL error.

**Comment by wb9688** [comment]
Then it's not our issue. Upgrade to a newer Android version

**Comment by mqus** [comment]
> Upgrade to a newer Android version.

That is not possible in many cases, so you could also send a nice email to the instance admin where you ask for enabling the right ciphers and at least tls 1.2.

**Comment by ale5000-git** [comment]
microG is also able to patch it with ProviderInstallerImpl.

**Comment by Younes-L** [comment]
I'm having the same problem on Android pie, newpipe 0.20.5 . 
I can add peertube.pontostroy.gq (follows 623 instances) to newpipe but not tube.4aem.com (follows 1049 instances)

Let me know if I should create a new issue

**Comment by mqus** [comment]
Please create a new issue with the exact error and how it occurred. This issue should not exist on android pie (9?), since it includes modern crypto.

**Comment by Younes-L** [comment]
@mqus  done #5115
